sensitivity-analysis/src/main/java/com/farao_community/farao/sensitivity_analysis/SystematicSensitivityInterface.java

```diff
@@ -144,6 +144,7 @@ public SystematicSensitivityResult run(Network network, Unit defaultUnit) {
             return result;
 
         } catch (SensitivityAnalysisException e) {
+            LOGGER.debug(e.getCause().getMessage());
```

Comment:
Prefer something such as:
`LOGGER.debug("Exception occured during sensitivity analysis", e)`
This way you will have the full stack trace.

Reply:
Ok, done!